### DISCUSSION TITLE
container/vxlan

### DISCUSSION THREAD

**Discussion by yshdzw** [isno/theByteBook#167] [General]
https://www.thebyte.com.cn/container/vxlan.html

**Comment by yshdzw** [comment]
”不过 VXLAN 有…… VLAN tag 的设计“。VXLAN ==> VLAN

**Reply by isno** [reply]
这是一篇历史文章。我删除了，但Google的搜索还在。
 你说的拼接错误以及VXLAN的内容我在这里重写了。
https://www.thebyte.com.cn/container/linux-vnet.html#%E7%BD%91%E7%BB%9C-vxlan

谢谢指正。
